### PR TITLE
Made Alert, Avatar and Badge Components

### PR DESCRIPTION
**Pull request by bhargaviRdy**
Added 3 components to the component library
image
image

Look into these  links for changes
https://vibe-ui-m30e6x4j8-bhargavirdy.vercel.app/docs/badge.html
https://vibe-ui-m30e6x4j8-bhargavirdy.vercel.app/docs/alerts.html
https://vibe-ui-m30e6x4j8-bhargavirdy.vercel.app/docs/avatar.html

### CODE REVIEW

File: components/alerts.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    width: 100%;
+    box-sizing: border-box;
+    border-radius: 0.3rem;
+    margin: 1rem 0;
+    padding: 1rem;
+    font-size: 1rem;
+    font-weight: 500;
+}
+.alert-icon{
+    margin-right: 0.625rem;
```
Comment (rkapoor10) [inline components/alerts.css:12]: you may use px values when it's less than 10px

File: components/avatars.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.avatar-xl{
+    width: 10rem;
+    height: 10rem;
+    font-size: 3rem;
+}
+.avatar-text{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    /* font-size: 2rem; */
```
Comment (rkapoor10) [inline components/avatars.css:28]: remove comments if not in use

**Review by harshitadaswani — APPROVED** [review]
The code looks good to merge

**Review by rkapoor10 — APPROVED** [review]
Minor changes. Good layout and base css. Go for merge.

**Pull request merged**